### DISCUSSION TITLE
[CLI] are flags supported?

### DISCUSSION THREAD

**Discussion by nyex-cell** [zieka/graphql-reshape#45] [Q&A] [answered]
Is there a way to use the cli with certain flags to bypass the prompts?

**Comment by zieka** [comment, marked as answer]
Currently, the cli only is usable in an interactive mode.  All of the directives and transformers are available in other packages in this mono repo if you would like to programmatically use them.

Adding flags for a non-interactive way to use the CLI would be a great way to contribute and would only require basic knowledge of the `commander` module.
